share/eds/pubsub.go

```diff
@@ -0,0 +1,83 @@
+package eds
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+// PubSubTopic hardcodes the name of the EDS floodsub topic.
+const PubSubTopic = "eds-sub"
```

Comment:
As per our discussion, we want to add semver and network id to the topic name. The version should be hardcoded, while chain id should be optionable

Reply:
I thought this would be done in follow-up but sure!